client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -46,7 +46,7 @@ class AuthorMappingPane extends PureComponent {
 		if ( numTargetUsers === 1 && numSourceUsers === 1 ) {
 			return this.props.translate(
 				'There is one author on your %(sourceType)s site. ' +
```

Comment:
Can we use "original" to be clearer about this?
```suggestion
				'There is one author on your original %(sourceType)s site. ' +
```